portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,45 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Casey Price</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="section" id="landing-content">
+      <h1>Casey Price</h1>
+      <p>Hi! I’m a rising junior at UC San Diego majoring in Computer Science. I’m currently working from home as a STEP Engineering Intern at Google and am serving as President of the ACM chapter at UC San Diego.</p>
```

Comment:
Can you break the line at 80 characters long?  IIRC it should render the same as long as you don't put in a <br> tag.